lib/logstash/outputs/kafka.rb

```diff
@@ -149,6 +149,20 @@ class LogStash::Outputs::Kafka < LogStash::Outputs::Base
   config :security_protocol, :validate => ["PLAINTEXT", "SSL", "SASL_PLAINTEXT", "SASL_SSL"], :default => "PLAINTEXT"
   # SASL client callback handler class
   config :sasl_client_callback_handler_class, :validate => :string
+  # The URL for the OAuth 2.0 issuer token endpoint.
+  config :sasl_oauthbearer_token_endpoint_url, :validate => :string
+  # (optional) The override name of the scope claim.
+  config :sasl_oauthbearer_scope_claim_name, :validate => :string, :default => 'scope'
+  # SASL login callback handler class
+  config :sasl_login_callback_handler_class, :validate => :string
+  # (optional) The duration, in milliseconds, for HTTPS connect timeout
+  config :sasl_login_connect_timeout_ms, :validate => :number, :default => 10000
```

Comment:
Don't we need to align the default value to kafka client [`sasl.login.connect.timeout.ms`](https://kafka.apache.org/31/generated/admin_client_config.html#adminclientconfigs_sasl.login.connect.timeout.ms) param? Same question for the `sasl.login.read.timeout.ms` and `sasl.login.retry.*` params.

Reply:
Seams reasonable. Appears to follow the pattern elsewhere in this file. Aligned the new options with defaults (or lack of defauts).

Reply:
Awesome, thank you so much!